inst/scripts/insert_image.R

```diff
@@ -0,0 +1,59 @@
+local({
+  txt_input = function(..., width = '100%') shiny::textInput(..., width = width)
+
+  blogfile = rstudioapi::getActiveDocumentContext()
```

Comment:
`rstudioapi::getSourceEditorContext()` would be a little more convenient for users.

Reply:
Changed. I agree, using this function makes more sense. I'm still new to `rstudioapi`